planyourcourses.com/docker-compose.course.yml

```diff
@@ -10,7 +10,7 @@ services:
         - "traefik.port=8080"
         - "traefik.enable=true"
         - "traefik.docker.network=external"
-        - "traefik.backend.loadbalancer.sticky=true"
+        - "traefik.backend.loadbalancer.stickiness=true"
         - "traefik.backend.loadbalancer.swarm=true"
```

Comment:
if our lb is swarm, the stickiness doesn't matter, the traefik will always proxy directly to the swarm lb which isn't sticky

Reply:
@vuzDylan made things 'sticky' and I don't really understand what that means. I was updating the syntax since 'sticky' is deprecated.